.buildkite/run.sh

```diff
@@ -6,13 +6,13 @@ set -eE
 
 function cleanup()
 {
-	# Rename sanitycheck junit xml for use with junit-annotate-buildkite-plugin
-	# create dummy file if sanitycheck did nothing
-	if [ ! -f sanity-out/sanitycheck.xml ]; then
-	   touch sanity-out/sanitycheck.xml
+	# Rename twister junit xml for use with junit-annotate-buildkite-plugin
+	# create dummy file if twister did nothing
+	if [ ! -f sanity-out/twister.xml ]; then
+	   touch sanity-out/twister.xml
 	fi
-	mv sanity-out/sanitycheck.xml sanitycheck-${BUILDKITE_JOB_ID}.xml
-	buildkite-agent artifact upload sanitycheck-${BUILDKITE_JOB_ID}.xml
+	mv sanity-out/twister.xml twister-${BUILDKITE_JOB_ID}.xml
+	buildkite-agent artifact upload twister-${BUILDKITE_JOB_ID}.xml
 
```

Comment:
Are these remaining `sanity-out` intentional, or does it need to be renamed too?